kedro/templates/project/hooks/post_gen_project.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os
+import shutil
+
+
+def parse_add_ons_input(add_ons_str):
+    """Parse the add-ons input string.
+
+    Args:
+        add_ons_str: Input string from prompts.yml.
+
+    Returns:
+        list: List of selected add-ons as strings.
+    """
+    if add_ons_str == "all":
+        return ["1", "2", "3", "4", "5"]
+    if add_ons_str == "none":
+        return []
+
+    # Split by comma
+    parts = add_ons_str.split(",")
```

Comment:
I find `parts` a pretty vague variable name, perhaps it could be "add_ons_choices" and then in the loop `for choice in add_ons_choice`?